.github/workflows/test-internal-prs.yml

```diff
@@ -13,7 +13,8 @@ concurrency:
 
 on:
   pull_request:
-    branches: [main, hotfix]
+    # branches: [main, hotfix]
+    branches: feat-expose-sb-types/main
```

Comment:
May need to revert the change before merge.

Reply:
Needs to reverted on the feature branch once this PR is merged